tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -32,7 +32,7 @@ def launch(self):
                 *self._settings_button_locator))))
 
     def switch_to_contacts_frame(self):
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
+        Wait(self.marionette).until(lambda m: self.apps.displayed_app.manifest_url == self.manifest_url)
```

Comment:
No, remember here we need to find a solution to distinguish between dialer and Contacts app and all the class herite from Base so we don't need to redefine it if it's  okay with our property https://github.com/mozilla-b2g/gaia/pull/31259#discussion-diff-37986457R154